### ISSUE TITLE
ExceptionInInitializerError on Android KitKat

### ISSUE THREAD

**Issue by gmazzo** [nulab/zxcvbn4j#66]
Since version `1.2.7` (it was working on `1.2.5`).

The logic for loading of `txt` classpath resources has changed, and now it's using `Thread.currentThread().getContextClassLoader()` though the `com.nulabinc.zxcvbn.matchers.ResourceLoader` class.

On old versions of Android, this `ClassLoader` points to `ClassLoader.getSystemClassLoader()` which doesn't hold any data related to the running app, and then the resources are not found.

This affects the `Dictionary` and `Keyboard` classes's static init block.

Our current workaround for this is to do in a `static init` method:
```java
    static {
        Thread thread = Thread.currentThread();
        ClassLoader prev = thread.getContextClassLoader();
        thread.setContextClassLoader(Zxcvbn.class.getClassLoader());

        // runs the static init block
        Dictionary.FREQUENCY_LISTS.size();
        Keyboard.ALL_KEYBOARDS.size();

        thread.setContextClassLoader(prev);
    }
```

**Comment by ricamgar** [comment]
Any updates on this?

**Comment by gmazzo** [comment]
At least the authros can open `ResourceLoader` to accept a class loader defaulting to `Thread.currentThread().getContextClassLoader()` to avoid breaking changes. 🤷‍♂️

**Comment by vvatanabe** [comment]
We are currently fixing it on pull request below
https://github.com/nulab/zxcvbn4j/pull/80

**Comment by vvatanabe** [comment]
@ricamgar I fixed it and released 1.3.2. Please check it!

**Comment by vvatanabe** [comment]
fixed 1.3.3
